bodhi/server/models.py

```diff
@@ -626,14 +626,17 @@ def get_pkg_pushers(self, branch, settings):
 
     def get_pkg_committers_from_pagure(self):
         """
-        Pull users who can commit and are watching a package in Pagure.
+        Pull users and groups who can commit on a package in Pagure.
 
-        Return two two-tuples of lists:
-        * The first tuple is for usernames.  The second tuple is for groups.
-        * The first list of the tuple is for committers. The second is for
-          watchers.
+        Returns a tuple with two lists:
+        * The first list is for usernames that have commit access. This list
+        will contain members from Pagure groups if `pagure_uses_fas_groups` is
+        set to True.
```

Comment:
Is this correct? The code below does not seem to consider the ```pagure_uses_fas_groups``` setting when adding users to the first list.